corehq/apps/reports/models.py

```diff
@@ -578,27 +579,33 @@ def by_domain_and_owner(cls, domain, owner_id, stale=True, **kwargs):
         return result
 
     @property
+    @memoized
     def all_recipient_emails(self):
         # handle old documents
         if not self.owner_id:
             return [self.owner.get_email()]
 
         emails = []
-        if self.send_to_owner:
-            if self.owner.is_web_user():
-                emails.append(self.owner.username)
-            else:
-                email = self.owner.get_email()
-                try:
-                    validate_email(email)
-                    emails.append(email)
-                except Exception:
-                    pass
+        if self.send_to_owner and self.owner_email:
+            emails.append(self.owner_email)
         emails.extend(self.recipient_emails)
         return emails
 
     @property
     @memoized
+    def owner_email(self):
+        if self.owner.is_web_user():
+            return self.owner.username
```

Comment:
yeah, just pulled it out so I could use it elsewhere.  I also wasn't crazy about the `except Exception` below, but not brave enough to fix it without testing that change alone.